src/messaging/ReliableMessageMgr.h

```diff
@@ -107,10 +107,11 @@ class ReliableMessageMgr
      *                              has been retransmitted so far (0 if it has
      *                              been sent only once with no retransmits,
      *                              1 if it has been sent twice, etc).
+     *  @param[in]   useRandom      Disable randomness such that the maximum value is used instead.
```

Comment:
Would it make sense to reverse the sense of this boolean and call it computeMaxPossible?  Either way.